processor/src/test/java/permissions/dispatcher/processor/data/Source.java

```diff
@@ -763,7 +763,7 @@ protected String getName() {
                     "import java.lang.String;",
                     "import permissions.dispatcher.PermissionUtils;",
                     "final class MyActivityPermissionsDispatcher {",
-                    "   private static final int REQUEST_SHOWCAMERA = 0;",
+                    "   private static final int REQUEST_SHOWCAMERA = 12;",
```

Comment:
I don't think this is reliable enough for us, because the order of tests might change over time, and this will become a huge hassle for us. Instead, let's consider opening up the `currentCode` field (maybe using `@VisibleForTesting` to highlight the purpose), and resetting its value to 0 before each test

Reply:
Umm yea..do you come up with any good idea? 🤔

Reply:
Opening `RequestCodeProvider`'s field should be enough:

```kotlin
object RequestCodeProvider {

    @VisibleForTesting
    val currentCode = AtomicInteger(0)

    fun nextRequestCode(): Int = currentCode.andIncrement
}
```

Then, inside our test class:

```kotlin
@Before
fun beforeEach() {
    RequestCodeProvider.currentCode = 0
}
```